elifepubmed/generate.py

```diff
@@ -364,6 +365,27 @@ def set_abstract(parent, poa_article, abstract_label_types):
         set_abstract_text(abstract_tag, '', '')
 
 
+def set_plain_language_summary(parent, article):
+    "set an OtherAbstract tag to include the digest as plain-language-summary"
+    tag_name = 'OtherAbstract'
+    if hasattr(article, 'digest') and article.digest:
+        # Pubmed allows <i> tags, not <italic> tags
+        tag_converted_digest = eautils.replace_tags(article.digest, 'italic', 'i')
+        tag_converted_digest = eautils.replace_tags(tag_converted_digest, 'bold', 'b')
+        tag_converted_digest = eautils.replace_tags(tag_converted_digest, 'underline', 'u')
+        tag_converted_digest = tag_converted_digest.replace('<p>', '').replace('</p>', '')
+        tag_converted_digest = etoolsutils.escape_unmatched_angle_brackets(
+            tag_converted_digest, utils.allowed_tags())
+        tagged_string = '<' + tag_name + '>' + tag_converted_digest + '</' + tag_name + '>'
```

Comment:
it's always fragile to build XML with string concatenation rather than rendering a data structure, but in this context I see no alternative given the library works with strings

Reply:
Turns out the `elifetools.xmlio` module has a function that can be used here that accepts the tag name and optional attributes. I refactored these to use it. I think I made that possible as part of developing another library that generates XML.